app/src/main/java/com/github/h3lp3rs/h3lp/MainPageActivity.kt

```diff
@@ -422,6 +428,20 @@ class MainPageActivity : AppCompatActivity(), OnRequestPermissionsResultCallback
         goToActivity(FirstAidActivity::class.java)
     }
 
+    /** Called when the user taps the professional portal  button */
+    fun goToProfessionalPortal(view : View) {
+        val db = databaseOf(Databases.PRO_USERS)
+        db.getObject(SignInActivity.userUid.toString(), ProUser::class.java).handle { _, err ->
+            if(err != null){
+                // If there is no proof of the status of the current user in the database, launch the verification process
+                goToActivity(VerificationActivity::class.java)
+                return@handle
+            }
```

Comment:
This certainly works but I worry about defining a user signed by the absence of error on the query.

Reply:
Nevermind it makes sense